viewing_party/party.py

```diff
@@ -1,23 +1,118 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
 
+    dict_movie={
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    for key in dict_movie:
+        if not dict_movie[key]:
+            return None
+    return dict_movie
+def add_to_watched(the_user_data, movie_watched):

```

Comment:
Nothing wrong at all but wanted to mention a quick thing about variable (or param) naming - no need to add an article to the name, so `user_data` would be a better choice.